addons/google_calendar/models/google_sync.py

```diff
@@ -127,7 +123,6 @@ def _from_google_ids(self, google_ids):
         return self.browse(self._event_ids_from_google_ids(google_ids))
 
     @api.model
-    @ormcache_context('google_ids', keys=('active_test',))
```

Comment:
I don't think this should change in the stable version. But it makes sense change this in master.

Reply:
I don't see any harm with the stable versions. Could you explain more?